dags/run_all_processors.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+from airflow.models import DAG
+from airflow.operators.latest_only_operator import LatestOnlyOperator
+import utils.helpers as helpers
```

Comment:
Should we start importing only the helper we need, in this case `create_processor_task`? Not sure how the import convention is applicable in this case.